code/modules/research/techweb/all_nodes.dm

```diff
@@ -254,6 +254,7 @@
 		"comp_index_assoc",
 		"comp_index_table",
 		"comp_length",
+		"comp_laserpointer",
```

Comment:
alphabetical, should be before comp_length

Reply:
missed that, sorry